src/version.inc

```diff
@@ -1,3 +1,3 @@
-#define LC0_VERSION_MAJOR 0
-#define LC0_VERSION_MINOR 14
-#define LC0_VERSION_PATCH 1
+#define LC0_VERSION_MAJOR 1
+#define LC0_VERSION_MINOR 0
+#define LC0_VERSION_PATCH 0
```

Comment:
Agreed, versions should always be separate commits. And besides that, any reason you're bumping to 1.0.0 instead of 0.15.0? We're very far from being even the slightest bit stable, imo, certainly from that stability which is implied by a 1.0.0

Reply:
I was being optimistic ;)